themes/falco-fresh/layouts/blog/section.rss.xml

```diff
@@ -21,6 +21,7 @@
       {{ with .Site.Author.email }}<author>{{.}}{{ with $.Site.Author.name }} ({{.}}){{end}}</author>{{end}}
       <guid>{{ .Permalink }}</guid>
       <description>{{ .Summary | html }}</description>
+      <content>{{ .Content | html }}</content>
```

Comment:
There is no `content` field in the RSS spec: https://validator.w3.org/feed/docs/rss2.html.

Again, the purpose of RSS feeds is not to transmit the entire contents of a blog. It is, rather, to notify RSS-compliant reader clients of updates.

Reply:
`content` belongs to `atom` xml ns, which is declared on the top of this file, shall we use it?